ema_pytorch/post_hoc_ema.py

```diff
@@ -283,7 +283,7 @@ def __init__(
         ema_model: Callable[[], Module] | None = None,
         sigma_rels: Tuple[float, ...] | None = None,
         gammas: Tuple[float, ...] | None = None,
-        checkpoint_every_num_steps: int = 1000,
+        checkpoint_every_num_steps: int | Literal['manual'] = 1000,
```

Comment:
how about for your usecase, just use a value of less than equal to 0 as `manual`

Reply:
`None` works too

Reply:
I considered both of those options, but I thought a string would be the most clear.  Just let me know what you prefer, though.  I'd be happy to change it to anything.

Reply:
if you think it is clear then let's do it

you're the one doing the training!